api-reference/10 UI Widgets/dxFileManager/1 Configuration/onErrorOccurred.md

```diff
@@ -6,28 +6,54 @@ EventForAction: dxFileManager.errorOccurred
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed when an error occurs.
 
 ##### param(e): Object
-<!-- Description goes here -->
+Information about the event that caused the function's execution.
 
 ##### field(e.component): {WidgetName}
-<!-- Description goes here -->
+The widget's instance.
 
 ##### field(e.element): dxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "widget" }
 
 ##### field(e.errorCode): Number
-<!-- Description goes here -->
+The error code.
 
 ##### field(e.errorText): String
-<!-- Description goes here -->
+The error message.
 
 ##### field(e.fileSystemItem): FileSystemItem
-<!-- Description goes here -->
+The processed file or directory.
 
 ##### field(e.model): Object
-<!-- Description goes here -->
+Model data. Available only if you use Knockout.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+The following error codes are supported:
+
+    NoAccess = 0,
```

Comment:
I think this information is more suitable in the description of the `errorCode` field.

Reply:
updated